README.md

```diff
@@ -1 +1,283 @@
-# backend
+
+# hot-coffee-sql
+
+## Learning Objectives
+
+- SQL
+- PostgreSQL
+- CRUD
+- ERD
```

Comment:
Nothing about ERD in the project, I guess it should be necessary for them to make ERD diagram and submit it with the work, because it really helps in organizing tables and relationships before impelmenting them